### PR TITLE
feature - new function for automatic reading mdr

### PR DESCRIPTION
**Pull request by thkoehler11**
- tiered menu instead dropdown for "add field"
- new bbmri logo

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (alexanderkiel) [inline .vscode/launch.json:1]: Please do not add IDE specific files.

File: src/config/env.js [hunk 2]
```diff
@@ -62,7 +64,8 @@
         "urn": "urn:mdr16:dataelement:27:1",
         "placeholder": "e.g. C25.1",
         "unit": "",
-        "valueSetUrl": "http://hl7.org/fhir/ValueSet/condition-code"
+        "valueSetUrl": "http://hl7.org/fhir/sid/icd-10-gm"
```
Comment (alexanderkiel) [inline src/config/env.js:67]: This change is already present in the master. Please merge the master in develop and rebase onto it.

File: src/environments/environment.ts [hunk 3]
```diff
@@ -5,7 +5,8 @@
 export const environment = {
   production: false,
   externalServices: (window as any).env.externalServices || {},
-  mdrConfig: (window as any).env.mdrConfig || {},
+  mdrConfigGBN: (window as any).env.mdrConfigGBN || {},
```
Comment (alexanderkiel) [inline src/environments/environment.ts:8]: Why can't we just use one mdrConfig like before?

File: src/config/env.js [hunk 4]
```diff
@@ -80,5 +83,67 @@
       "urn:mdr16:dataelement:24:1",
       "urn:mdr16:dataelement:13:1"
     ]
-  };
+  }
+  window["env"]["mdrConfigCCDG"] =
```
Comment (alexanderkiel) [inline src/config/env.js:87]: Could we just have one env.js for GBN and one for BBMRI-ERIC?

File: src/config/env.js [hunk 5]
```diff
@@ -11,19 +11,21 @@
     "brokerUrl": "http://localhost:8080/broker",
     "sampleLocatorUrl": "http://localhost:8085",
     "negotiatorUrl": "https://negotiator.bbmri-eric.eu",
-    "terminologyServerUrl": "https://r4.ontoserver.csiro.au/fhir"
+    "terminologyServerUrl": "https://samplelocator.bbmri.de/icd10"
```
Comment (alexanderkiel) [inline src/config/env.js:14]: This change is already present in the master. Please merge the master in develop and rebase onto it.

File: src/app/component/work-in-progress/work-in-progress.component.html [hunk 6]
```diff
@@ -1,7 +1,7 @@
 <div class="work-in-progress-info bbmri-transition">
   <div class="bbmri-container">
     <h4>
-      +++ Sample Locator +++ search for biosamples and associated data across multiple biobanks +++
+      +++ BBMRI Locator +++ search for biosamples and associated data across multiple biobanks +++
```
Comment (alexanderkiel) [inline src/app/component/work-in-progress/work-in-progress.component.html:4]: We can't simply change this. Has to be a config.

File: src/app/component/header/header.component.html [hunk 7]
```diff
@@ -6,7 +6,7 @@
       <div class="logo-main col-md-5">
         <a routerLink="/" title="samplelocator.bbmri.de">
           <!--suppress HtmlUnknownTarget -->
-          <img src="assets/img/GBN_CMYK_schwarz.svg" alt="GBA-LOGO" class="gba-logo"/>
+          <img src="assets/img/BBMRI-ERIC-gateway-for-health.svg" alt="BBMRI-ERIC-LOGO" class="bbmri-logo"/>
```
Comment (alexanderkiel) [inline src/app/component/header/header.component.html:9]: We can't simply change this. Has to be a config.

**Review by alexanderkiel — CHANGES_REQUESTED** [review]

**Review by lablans — DISMISSED** [review]
Some optional comments included

**Review by alexanderkiel — APPROVED** [review]

**Pull request merged**